Dalamud/Game/Inventory/GameInventoryItem.cs

```diff
@@ -101,11 +104,17 @@ static GameInventoryItem()
     public ReadOnlySpan<ushort> Materia => new(Unsafe.AsPointer(ref Unsafe.AsRef(in this.InternalItem.Materia[0])), 5);
 
     /// <summary>
-    /// Gets the array of materia grades.
+    /// Gets the array of materia grades as bytes.
     /// </summary>
-    public ReadOnlySpan<byte> MateriaGrade =>
+    public ReadOnlySpan<byte> MateriaGradeBytes =>
         new(Unsafe.AsPointer(ref Unsafe.AsRef(in this.InternalItem.MateriaGrade[0])), 5);
```

Comment:
Mark this as `private` and place a `// TODO` for API 10 to fix the type. I'd like to avoid adding a new API that's just going to be removed in the very near future.

Reply:
Prefer to use `[Api10ToDo(Api10ToDoAttribute.DeleteCompatBehavior)]` so that it's easier to track down what to change at a later date.